### PR TITLE
Added TH playbook

### LINKED ISSUE #239: Playbook: Thailand (TH)
### Country

Thailand (TH)

### eAIP

_No response_

### Steps to add a new playbook

You can check [playbooks](https://github.com/drph4nt0m/avbot-charts/tree/main/playbooks) for examples on chart playbooks and [axios engine](https://github.com/drph4nt0m/avbot-charts/blob/main/app/engines/axios.js) on how the features in the playbook are used by the axios scraping engine.

Tasks summary:
-[ ] Fork & clone AvBot Charts
-[ ] Create playbook for the country in playbooks
-[ ] Add tests for the country in tests/getChart.test.js (tests should be in alphabetical order of the country code)
-[ ] Push changes and submit a pull request
-[ ] If you need any help, reach out to us on our [Discord server](http://link.avbot.in/support).

Are you ready to work on this issue? Let us know, and we will assign it to you.

Happy Coding!

### PR DESCRIPTION
**Pull request by akgupta0777**
# Description

Added Thailand Playbook.

Fixes #239 

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [X] Test A
- [X] Test B

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation/readme
- [X] My changes generate no new warnings
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by akgupta0777** [comment]
**Results of TH Playbook**

image

**Comment by drph4nt0m** [comment]
Hey @akgupta0777, also please add tests in tests/getChart.test.js

**Comment by akgupta0777** [comment]
Sure will add tests also.

**Comment by akgupta0777** [comment]
@drph4nt0m I already formatted my code with prettier still it fails why ?

**Comment by drph4nt0m** [comment]
Not the GitHub actions. Ignore those, there seems to be some miss configuration by me. Check the tests/getChart.test.js and add tests for TH

**Comment by akgupta0777** [comment]
@drph4nt0m please review

**Comment by akgupta0777** [comment]
can you please add the hacktoberfest-accepted label if you are willing to merge this 😅

File: tests/getAirport.test.js [hunk 1]
```diff
@@ -7,6 +7,12 @@ test('IN', async () => {
   expect(airport).toMatchObject({ ident: 'VABB' });
 });
 
+test('TH', async () => {
```
Comment (drph4nt0m) [inline tests/getAirport.test.js:10]: This test is not need. Remove it please.

**Review by drph4nt0m — CHANGES_REQUESTED** [review]
Please work on the comment and push the changes again

File: tests/getAirport.test.js (hunk 1 above)
Comment (akgupta0777) [reply tests/getAirport.test.js:10]: ok

**Comment by akgupta0777** [comment]
@drph4nt0m Will keep in mind not to add tests for getAirport.test.js

**Comment by drph4nt0m** [comment]
> can you please add the hacktoberfest-accepted label if you are willing to merge this 😅

@akgupta0777 since the whole repo is tagged hacktoberfest, I dont need to add that label to PRs.

Thanks for your contribution. Looking forward to more PRs from you.

**Pull request merged**